manifests/n/Nonpolynomial/IntifaceCentral/2.6.0/Nonpolynomial.IntifaceCentral.locale.en-US.yaml

```diff
@@ -0,0 +1,41 @@
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.6.0.schema.json
+PackageIdentifier: Nonpolynomial.IntifaceCentral
+PackageVersion: 2.6.0
+PackageLocale: en-US
+Publisher: Nonpolynomial Labs
+PublisherUrl: https://nonpolynomial.com
+PublisherSupportUrl: https://discuss.buttplug.io
+PackageName: Intiface Central
+PackageUrl: https://intiface.com/central
+License: GNU General Public License v3.0
+LicenseUrl: https://github.com/intiface/intiface-central/blob/main/LICENSE.md
+ShortDescription: Intiface Central is an open-source, cross-platform application that acts as a hub for intimate haptics/sensor hardware access.
+Description: |-
+  Intiface Central is a frontend application for the BSTC Library, for Desktop (Win/macOS/Linux) and Mobile (Android/iOS). It can be used with programs such as:
+    - Intiface Game Haptics Router
+    - ScriptPlayer
+    - Toy Playground
+    - Many other apps listed on the Awesome App/Game list!
+  
+  For users, it provides simple, friendly capabilites for managing, connecting, customizing devices.
+  
+  For developers, it allows their application to connect to and control adult toys, without having to worry about constantly updating the underlying libraries or dealing with bugs and crashes in a difficult-to-debug cross langauge environment.
+Moniker: intifacecentral
+Tags:
+- intiface
+- bluetooth
+Documentations:
+  - DocumentLabel: Documentation
+    DocumentUrl: https://docs.intiface.com/docs/intiface-central
+  - DocumentLabel: FAQ 
+    DocumentUrl: https://intiface.com/central/#FAQ
+ReleaseNotes: |-
+  Update to Buttplug v8/Intiface Engine v3
```

Comment:
This likely violates content policies - If this engine name appears anywhere in the application (and I believe it does, in the about page), it will probably be flagged and we likely won't be able to add this application

Reply:
Thank you for the clarification and feedback.  ^_^